src/objects/npe03__Recurring_Donation__c.object

```diff
@@ -448,6 +448,11 @@
                     <default>false</default>
                     <label>Closed</label>
                 </value>
+                <value>
```

Comment:
This will be deployed as Inactive to all current customers (no matter what their RD2 state is). The other 3 picklist values were deployed when the field was first deployed so they are already active for all customers. Unless we're going to add unpackaged metadata to activate this when RD2 is enabled, the documentation will need to be updated to show customers how to manually activate this new picklist value.

Reply:
@sam-knox  ^

Reply:
Does that mean I should add the Status field to the rd2 post config?

Reply:
Yes if there is consensus that we want to auto-activate this new picklist value when RD2 is enabled.

Reply:
Just want to make sure I understand:

1. For new or existing customers who enable RD2 after we merge this PR, there is no action needed. The new picklist value is added and activated automatically.
2. For existing customers who already enabled RD2 prior to this merge, they would need to add the new picklist value manually.

Is that correct?

Reply:
That is correct from my understanding

Reply:
With regard to customers already having ERD enabled, they need to activate the Paused picklist value manually.

Reply:
@RadSFDeveloper - Ok, so existing customers will get "Paused" as a value in the picklist field, they just need to activate it?

Reply:
As part of regular NPSP push upgrades?

Reply:
I believe so. @force2b , you are the expert on the metadeploy and updates, can you please confirm?
CC @ravensisco

Reply:
I want to correct this statement from above:
> For new or existing customers who enable RD2 after we merge this PR, there is no action needed. The new picklist value is added and activated automatically.

When a new picklist value is added to an existing field in a managed package, it will be added to all customer orgs as **Inactive** regardless of whether they have enabled RD2 or not. Though, reading the [ISV documentation](https://developer.salesforce.com/docs/atlas.en-us.packagingGuide.meta/packagingGuide/packaging_component_behavior.htm) it clearly states that "Picklist field values can be added or deleted in the developer’s organization. Upon upgrade, no new values are installed." That is not my memory of observed behavior, so it might be worth testing. So, either they're getting "Paused" as an inactive picklist value on the field or "Paused" is not being added at all. 

We can choose to activate (and add if necessary) the new Paused picklist value for customers who enable RD2 in the future by adding this field and the picklist values to the unpackaged metadata, marked as "Active". If that is not done, then the Paused picklist value will be inactive (or possibly not exist) for everyone until they manually activate it.